pkgs/stdenv/generic/default.nix

```diff
@@ -223,7 +223,7 @@ let
         # include it in the result, it *is* available to nix-env for queries.
         meta = { }
             # If the packager hasn't specified `outputsToInstall`, choose a default,
-            # namely `p.bin or p.out or p`;
+            # namely `lib.getBin p`;
```

Comment:
You mean [there](https://github.com/abbradar/nixpkgs/blob/search-path-fixes/pkgs/stdenv/generic/default.nix#L235)? That's because the logic is a bit different -- if not `bin` nor `out` is found it iterates by `outs` instead of just getting the default (unspecified output).